wowup-electron/src/app/pages/my-addons/my-addons.component.html

```diff
@@ -227,6 +227,9 @@ <h1>{{ "COMMON.SEARCH.NO_ADDONS" | translate }}</h1>
     <button mat-menu-item (click)="onShowFolder(listItem.addon)">
       {{ "PAGES.MY_ADDONS.ADDON_CONTEXT_MENU.SHOW_FOLDER" | translate }}
     </button>
+    <button mat-menu-item (click)="onVisitProvider(listItem.addon)">
+      {{ "DIALOGS.ADDON_DETAILS.VIEW_ON_PROVIDER_PREFIX" | translate }} {{listItem.addon.providerName}}
```

Comment:
Can you add this translation to `en.json` and then run `npm run i18n` to sync the translations?

Reply:
Its already in the translations as its used on the addon detail popup also.